addons/product/static/src/scss/product_form.scss

```diff
@@ -1,3 +1,7 @@
 .oe_title .o_favorite i.fa {
     font-size: inherit;
 }
+
+.o_field_image {
+    background-color: transparent;
+}
```

Comment:
We cannot do that since you will alter all elements with that class instead of just altering the image in the kanban view of product catalog. 😕 

Why not adding `.bg-transparent` in the image instead of adding a css style?